src/clients/github.js

```diff
@@ -143,20 +124,20 @@ export function createGitHubRepository(projectName, projectDescription, token) {
  * @param projectName
  */
 export function initAddCommitAndPush(username, projectName, isTwoFactorAuth) {
-  winston.log('verbose', 'initAddCommitAndPush', { username, projectName });
-  winston.log('info', 'Pushing all files to the new git repository...');
+  winston.log('verbose', 'initialize github repo, create repo and push to repo', { username, projectName });
```

Comment:
Keep the method name for this one, but change it to `debug` level. We are already giving the user information at the `info` level, and the verbose level seems a bit redundant.